theano/sandbox/rng_mrg.py

```diff
@@ -11,33 +11,36 @@
 import numpy
 from six.moves import xrange
 
-from theano import Op, Apply, shared, config, Variable, Out
+from theano import Op, Apply, shared, config, Variable
 from theano import gradient, function
 from theano import tensor
-from theano.tensor import (raw_random, TensorType, as_tensor_variable,
-                           get_vector_length, cast, opt, scal)
+from theano.tensor import (TensorType, as_tensor_variable, get_vector_length,
+                           cast, opt, scal)
 from theano.tensor import sqrt, log, sin, cos, join, prod
 from theano.compile import optdb
 from theano.gof import local_optimizer
 from . import multinomial
 
-import theano.sandbox.cuda
 from theano.sandbox.cuda import GpuOp
-if theano.sandbox.cuda.cuda_available:
-    from theano.sandbox.cuda import (CudaNdarrayType,
-                                     float32_shared_constructor)
-
 from theano.sandbox.gpuarray.basic_ops import GpuKernelBase, Kernel
 from theano.sandbox.gpuarray.type import GpuArrayType
 from theano.sandbox.gpuarray.fp16_help import write_w
+from theano.sandbox.gpuarray.opt import (register_opt as register_gpua,
+                                         host_from_gpu as host_from_gpua)
+import theano.sandbox.cuda
```

Comment:
Is there a reason why you moved the import from `theano.sandbox.cuda` downwards? If possible change the order of the imports so that each section of imports (system libraries, generic libraries and framework-related libraries) is alphabetically ordered.